### PR TITLE
[tensorflow][inference] add dockerfiles for tfs 2.7

### PR DESCRIPTION
**Pull request by hsl89**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right.

### CODE REVIEW

File: tensorflow/inference/docker/2.7/py3/Dockerfile.e3.cpu [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    "awscli<2" \
+    boto3 \
+    cython==0.29.21 \
+    gevent==21.1.1 \
+    requests==2.25.1 \
+    grpcio==1.34.1 \
+    protobuf==3.14.0 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api=="2.7.0rc1"
```
Comment (arjkesh) [inline tensorflow/inference/docker/2.7/py3/Dockerfile.e3.cpu:71]: can you update this to 2.7.0

File: tensorflow/inference/docker/2.7/py3/cu112/Dockerfile.e3.gpu [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    "awscli<2" \
+    boto3 \
+    cython==0.29.21 \
+    gevent==21.1.1 \
+    requests==2.25.1 \
+    grpcio==1.34.1 \
+    protobuf==3.14.0 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api-gpu=="2.7.0rc1"
```
Comment (arjkesh) [inline tensorflow/inference/docker/2.7/py3/cu112/Dockerfile.e3.gpu:116]: can you update this to 2.7.0?

**Comment by arjkesh** [comment]
Can you make this change https://github.com/aws/deep-learning-containers/blob/25264738b01c373111bba07a9a26b2ebb6dbe7b5/test/dlc_tests/sanity/test_pre_release.py#L331 on this PR as well? This will resolve dependency check issue

File: tensorflow/buildspec.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK tensorflow
-version: &VERSION 2.6.0
-short_version: &SHORT_VERSION 2.6
+version: &VERSION 2.7.0 
+short_version: &SHORT_VERSION 2.7
```
Comment (arjkesh) [inline tensorflow/buildspec.yml:5]: nit: looks like some whitespace got added here, let's just remove

**Review by tejaschumbalkar — COMMENTED** [review]
LGTM

**Review by tejaschumbalkar — APPROVED** [review]

File: tensorflow/inference/docker/2.7/py3/Dockerfile.e3.cpu [hunk 4]
```diff
@@ -0,0 +1,111 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3.8
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.8.9
+ARG TFS_SHORT_VERSION=2.7
```
Comment (tejaschumbalkar) [inline tensorflow/inference/docker/2.7/py3/Dockerfile.e3.cpu:10]: nit: unused `TFS_SHORT_VERSION`

**Review by tejaschumbalkar — APPROVED** [review]

**Pull request merged**